x/evm/keeper/msg_server.go

```diff
@@ -48,18 +50,29 @@ func (k *Keeper) EthereumTx(goCtx context.Context, msg *types.MsgEthereumTx) (*t
 	}
 
 	defer func() {
-		if tx.Value().IsInt64() {
-			telemetry.SetGauge(
-				float32(tx.Value().Int64()),
-				"tx", "msg", "ethereum_tx",
-			)
-		}
-
 		telemetry.IncrCounterWithLabels(
-			[]string{types.ModuleName, "ethereum_tx"},
+			[]string{"tx", "msg", "ethereum_tx", "total"},
 			1,
 			labels,
 		)
+
+		if response.GasUsed != 0 {
+			telemetry.IncrCounterWithLabels(
+				[]string{"tx", "msg", "ethereum_tx", "gas_used", "total"},
+				float32(response.GasUsed),
+				labels,
+			)
+
+			gasUsed := sdk.NewDec(int64(response.GasUsed))
+			gasRatio, err := gasUsed.QuoInt64(int64(tx.Gas())).Float64()
```

Comment:
add comment that we want to observe which senders define a `gas limit >> gas used`

Reply:
also, add comment: tx gas limit is always > 0